static/js/lightbox.js

```diff
@@ -21,20 +21,29 @@ export class PanZoomControl {
     constructor(container) {
         this.container = container;
         this.panzoom = panzoom(this.container, {
-            disablePan: true,
-            disableZoom: true,
-            cursor: "auto",
+            smoothScroll: false,
+            maxZoom: 5,
+            minZoom: 0.2,
```

Comment:
Why these 0.2/5 limits?  With smaller images I definitely would prefer to be able to go 20x.